### PR TITLE
updated README in styling for cos function, in ref to link.…

### PR DESCRIPTION
**Pull request by Dylan-Brown**
https://github.com/AnalyticalGraphicsInc/cesium/pull/4578

The PR in the cesium repo should probably be merged before this one, of course.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Thanks @Dylan-Brown!

Please add an Acknowledgments section and add yourself like this: https://github.com/AnalyticalGraphicsInc/3d-tiles#acknowledgments

File: Styling/README.md [hunk 1]
```diff
@@ -488,6 +489,18 @@ Returns the absolute value of the number.
...
 
+#### cos 
+
+`cos(value : Number) : Number`
+
+Returns the cosine value of the number (in radians).
+
+```json
+{
+    "show" : "cos(${temperature}) > 1.0"
```
Comment (pjcozzi) [inline Styling/README.md]: `cos` will never be greater than `1` so perhaps use a different example.

Comment (Dylan-Brown) [inline Styling/README.md]: updated to 0.0

**Comment by Dylan-Brown** [comment]
Done!

**Comment by lilleyse** [comment]
Nice!

**Comment by pjcozzi** [comment]
Thanks again @Dylan-Brown!

**Pull request merged**